build-logic/src/main/kotlin/org.sonarsource.iac.integration-test.gradle.kts

```diff
@@ -16,6 +16,9 @@ val integrationTestTask =
     tasks.register<Test>("integrationTest") {
         description = "Runs integration tests."
         group = "verification"
+        inputs.dir("$rootDir/its/sources")
+        inputs.property("SQ version", System.getProperty("sonar.runtimeVersion", "LATEST_RELEASE"))
```

Comment:
Funny, I was trying to get the property also for the Plugin ITs, but failed so I used env instead. So I learned something new. 😆